HousingRepairsOnlineApi/Helpers/CancelAppointmentStatus.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace HousingRepairsOnlineApi.Helpers;
+
+public enum CancelAppointmentStatus
+{
+    Found,
```

Comment:
The first enum entry (or the one assigned a value of '0') is the default value is no value is set.
It would be best to have a default value which does not indicate an actual _result_ status, i.e. `Found`, 'NotFound', etc.

I would suggest adding an `Unknown` entry as the initial/default item/value:
```suggestion
    Unknown,
    Found,
```